etc/inc/disable-programs.inc

```diff
@@ -1107,6 +1108,7 @@ blacklist ${HOME}/.sbt
 blacklist ${HOME}/.scorched3d
 blacklist ${HOME}/.scribus
 blacklist ${HOME}/.scribusrc
+blacklist ${HOME}/.sendgmail.*
```

Comment:
```suggestion
blacklist ${HOME}/.sendgmail.json
```

> https://github.com/google/gmail-oauth2-tools/tree/master/go/sendgmail

That page only mentions ~/.sendgmail.json in the root of the home diectory.

Is the wildcard intended to match anything else specifically?

If not, I think it would be better to be explicit.

Reply:
There are per-account configs as well like `.sendgmail.some@email.json` created with `-setup` option. https://github.com/google/gmail-oauth2-tools/blob/e6adc9e1d2fb66344295449e73c71b9008fddd51/go/sendgmail/paths.go#L38